### PR TITLE
#Issue10 - Added functionality for zoom-in and zoom-out

### LINKED ISSUE #9: As a JavaFX user, I want to add zoom-in, zoom-out functionality, so that I can enlarge or reduce the size of text when needed
Sprint-1

### LINKED ISSUE #10: As part of Issue 9, write a functionality to create zoom in, zoom out option
Sprint-1 
Points - 5

### LINKED ISSUE #11: As a JavaFX developer, I want doxygen formatted code, so that the design document can be generated easily whenever needed

### PR DESCRIPTION
**Pull request by paidikondala**
As part of issue 10, implemented zoom-in and zoom-out functionalities

### CODE REVIEW

**Review by aanuhya31 — APPROVED** [review]
I see no issues with the code.

**Review by sireeshanalabothu — APPROVED** [review]

File: src/main/java/edu/wright/airviewer2/AIRViewerController.java [hunk 1]
```diff
@@ -575,6 +605,72 @@ public void initialize(URL url, ResourceBundle rb) {
...
+		scrollPane.setStyle(scrollCssLayout);
+		scrollPane.setVbarPolicy(ScrollBarPolicy.AS_NEEDED);
+		scrollPane.setHbarPolicy(ScrollBarPolicy.AS_NEEDED);
+		scrollPane.setContent(imageView);
+
+	}
+
+	@FXML
+	private void zoomIn() {
+		//zoom.set(zoom.get() * 1.1);
```
Comment (aanuhya31) [inline src/main/java/edu/wright/airviewer2/AIRViewerController.java]: Please remove these comments

File: src/main/java/edu/wright/airviewer2/AIRViewerController.java [hunk 2]
```diff
@@ -575,6 +605,72 @@ public void initialize(URL url, ResourceBundle rb) {
...
+		new ZoomInZoomOut().zoomIn(zoom);
+
+	}
+
+	@FXML
+	private void zoomOut() {
+		//zoom.set(zoom.get() / 1.1);
```
Comment (aanuhya31) [inline src/main/java/edu/wright/airviewer2/AIRViewerController.java]: Please remove this too

**Review by sireeshanalabothu — APPROVED** [review]
review completed and changes looks good.

File: src/main/java/edu/wright/airviewer2/ZoomInZoomOut.java [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+	  * \note         
+	  *          
+	  * \param[in]  index           get the pdf current page count
+	  * 
+	  * \param[in]  model           get the wrapped model with model view
+	  * 
+	  * \param[out] void      
+	  * 
+	  * 
+	  */
```
Comment (aanuhya31) [inline src/main/java/edu/wright/airviewer2/ZoomInZoomOut.java:87]: Doxygen comments looks good

**Review by aanuhya31 — CHANGES_REQUESTED** [review]

File: src/main/java/edu/wright/airviewer2/AIRViewerController.java (hunk 1 above)
Comment (paidikondala) [reply src/main/java/edu/wright/airviewer2/AIRViewerController.java]: Removed the comments

**Review by aanuhya31 — APPROVED** [review]
Everything looks good now

**Pull request merged**